include/command.h

```diff
@@ -200,6 +200,29 @@ class Command
         void update_key(std::string old_key,
                         std::string new_key);
 
+        /*!
+        *   \brief Set address and port for command
+        *          to be executed on
+        *   \param address Address of database
+        *   \param port Port of database
+        */
+        void set_exec_address_port(std::string address,
+                                   uint64_t port);
+
+        /*!
+        *   \brief Get address that command will be
+        *          to be executed on
```

Comment:
Maybe we can indicate what would be returned if the ``set_exec_address_port()`` method had never been called?  If we go with the default value of a ``std::string``, maybe we should say an empty string is returned.

Reply:
yep ill add that an empty string is returned if address isn't set